tests/AbstractQueryProvider.php

```diff
@@ -57,7 +57,7 @@
 use TheCodingMachine\GraphQLite\Utils\Namespaces\NamespaceFactory;
 use UnitEnum;
 
-abstract class AbstractQueryProviderTest extends TestCase
+abstract class AbstractQueryProvider extends TestCase
```

Comment:
Use `TestCase` suffix - it's becoming a de-facto standard for these abstract test cases:

https://github.com/rectorphp/rector-phpunit/tree/main/rules-tests/PHPUnit100/Rector/MethodCall/RemoveSetMethodsMethodCallRector#L11
https://github.com/laravel/framework/blob/978ee8887801d4000a73591c6ca19805bf4c229a/tests/Integration/Database/MariaDb/MariaDbTestCase.php#L7
https://github.com/symfony/symfony/blob/3f2ed0fc4c41744991113185a619baf5d2f7590f/src/Symfony/Bridge/Twig/Tests/Extension/AbstractTableLayoutTestCase.php#L17